static/css/pages/privacy-policy.css

```
@@ -0,0 +1,223 @@
+  --pp-surface-glass: rgba(30, 32, 36, 0.75);
+}
+
+/* Navbar */
+.pp-navbar {
+
+  /* ensures the nav stays at the top of the page */  
+  position: sticky;
+  top: 0;
+  z-index: 999;  
+
+  display: flex;
```

Stylelint (17.14.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)